lib/config-schema.js

```diff
@@ -0,0 +1,8 @@
+'use babel'
```

Comment:
Yeah, I think this is nice. As long as you're moving it into a separate file, what do you think of having it be a simple JSON file, instead of javascript? We've talked in the past about how we wish  we had required it to be like that, so that we could load a package's config without having to run any of the package's code.